aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -312,18 +348,34 @@ void raw_cudnn_convolution_forward_out(
       .build());
   }
 
+  c10::optional<cudnn_frontend::Operation> relu_op;
+  if (kReluFused) {
+    // TODO: can we assign the result back into conv_output and get rid of after_relu?
+    relu_op.emplace(cudnn_frontend::OperationBuilder(CUDNN_BACKEND_OPERATION_POINTWISE_DESCRIPTOR)
+      .setxDesc(bias.has_value() ? sum_conv_bias_op.value().getOutputTensor() : conv_op.getOutputTensor())
+      .setyDesc(getTensorDescriptor(after_relu.value(), 'f', getAlignment(after_relu.value())))
+      .setpwDesc(getPointWiseReluDescriptor(getCudnnDataType(after_relu.value())))
+      .build());
+  }
+
   auto requant_op = cudnn_frontend::OperationBuilder(CUDNN_BACKEND_OPERATION_POINTWISE_DESCRIPTOR)
-    .setxDesc(bias.has_value() ? sum_conv_bias_op.value().getOutputTensor() : conv_op.getOutputTensor())
+    .setxDesc(bias.has_value() ? (kReluFused ? relu_op.value().getOutputTensor() : sum_conv_bias_op.value().getOutputTensor())
+                               : (kReluFused ? relu_op.value().getOutputTensor() : conv_op.getOutputTensor()))
```

Comment:
these are getting complicated, maybe we can have a variable for `bias.has_value() ? sum_conv_bias_op.value().getOutputTensor() : conv_op.getOutputTensor()` and reuse it here, this is also used in L355? (we can first check has_bias() and then kReLUFused I think)

Reply:
ah yes I think that will work. I thought getOutputTensor() returned a tensor, but it looks like it returns ManagedOpaqueDescriptor, which is just a shared_ptr, so storing it should be fine efficiency-wise. i'll make the change